### PR TITLE
#165562029 upgrade Django to version 2.2

### PR DESCRIPTION
**Pull request by Amoswachira**
#### Description
 Upgrade Django form version 2.1  to  Django version 2.2 to fix security vulnerability issues, upgrade Django rest framework  and Django extensions

#### Type of change
- [x] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165562029](https://www.pivotaltracker.com/n/projects/2326460/stories/165562029)

### CODE REVIEW

**Review by a-braham — APPROVED** [review]
LGTM

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+Django==2.2
 django-cors-middleware==1.2.0
-django-extensions==1.7.1
-djangorestframework==3.4.4
+django-extensions==2.1.6
+djangorestframework==3.9.2
 isort==4.3.16
 lazy-object-proxy==1.3.1
 mccabe==0.6.1
+pycodestyle==2.5.0
```
Comment (IssaIan) [inline requirements.txt]: We already have pylint. We do not need pycodestyle.

**Review by IssaIan — APPROVED** [review]
LGTM

File: requirements.txt (hunk 1 above)
Comment (Amoswachira) [reply requirements.txt]: @IssaIan i have made the changes

**Pull request merged**